### PR TITLE
Feature/tm absence

### PR DESCRIPTION
**Pull request by JaseZiv**
addressing [#352](https://github.com/JaseZiv/worldfootballR/issues/352)

Still need to update vignettes, run local check

EDIT: Now completed

### CODE REVIEW

**Comment by JaseZiv** [comment]
The new function `tm_get_player_absence()` grabs this:

image


and returns it as so in an R data frame:

image


Happy with that @tonyelhabr?

File: vignettes/extract-transfermarkt-data.Rmd [hunk 1]
```diff
@@ -297,7 +297,7 @@ dplyr::glimpse(hazard_injuries)
 #----- for multiple players: -----#
 # # can make use of a tm helper function:
 # burnley_player_urls <- tm_team_player_urls(team_url = "https://www.transfermarkt.com/fc-burnley/startseite/verein/1132/saison_id/2021")
-# # then pass all those URLs to the tm_player_bio
+# # then pass all those URLs to the tm_player_injury_history
```
Comment (JaseZiv) [inline vignettes/extract-transfermarkt-data.Rmd:300]: drive by fix

**Review by tonyelhabr — APPROVED** [review]
didn't realize you were a Spurs fan (judging by the players chosen for examples) 😄

**Comment by JaseZiv** [comment]
> didn't realize you were a Spurs fan (judging by the players chosen for examples) 😄

Haha not a Spurs fan, but definitely a Big Ange fan (and also 'Romero' was the first name with suspensions that came to mind to type in transfermarkt's search bar)!

**Pull request merged**